content/articles/a-beginners-guide-on-how-to-become-a-blockchain-developer/index.md

```diff
@@ -0,0 +1,181 @@
+
+2. Ethereum blockchain
+
+The Ethereum blockchain is a decentralized and open-source smart contracts platform. The smart contract functionality that enables Dapps to be built on it has made it the most actively used blockchain platform. It currently uses PoW as its consensus mechanism but aiming to replace it with [Proof-of-Stake (PoS)](https://en.wikipedia.org/wiki/Proof_of_stake) due to the high energy consumption of PoW. 
+
+It is also the blockchain technology that powers the popular cryptocurrency, ether (ETH).
+
+Learn more about it [here](https://ethereum.org/en/).
+
+3. Cardano blockchain
+
+It is a decentralized and open-source blockchain platform that utilizes Proof-of-Stake (PoS) as its consensus mechanism. It is the blockchain platform that powers the popular cryptocurrency known as Ada. 
+Unlike the Ethereum blockchain, Cardano is developed through evidence-based methods and peer-reviewed research. This means that proposed methods have to undergo academic-type research before being implemented.
+
+This blockchain technology is built on the Haskell programming language commonly used in the banking and financial industry. 
+
+#### Note
+
+Unless you have a really good reason, then you should use the Ethereum blockchain as it has the biggest developer community for building blockchain-based applications. As a developer, you wouldn't want to get stuck and not have a community to be able to ask questions.
+
+### Libraries and frameworks for blockchain development
+
+1. [Web3.js](https://web3js.readthedocs.io/en/v1.3.4/)
+
+This is a collection of libraries that allows you to interact with the Ethereum node either remotely or locally using HTTP, Websocket, or IPC.
+
+2. [React.js](https://reactjs.org/)
+
+This is an open-source javascript library that enables programmers to build beautiful user interfaces. You can use React.js to build those beautiful user interfaces on the front-end.
+
+3. [Ethers.js](https://docs.ethers.io/v5/)
+
+It is a collection of libraries that allows you to interact with the Ethereum blockchain and its ecosystem.
+
+4. [Metamask](https://metamask.io/)
+
+It is a browser extension that allows a user to access their Ethereum wallets on the Ethereum blockchain. Users can buy, store, and swap ether tokens using this library.
+
+5. [Node.js](https://nodejs.org/en/)
+
+It is a back-end JavaScript runtime environment that executes javascript code outside a web browser. Decentralized applications usually use node.js to interact with smart contracts and public nodes.
+
+6. [Ganache](https://www.trufflesuite.com/ganache)
+
+It is a library developers can use for Ethereum blockchain development. It enables developers to develop, deploy, and test decentralized applications in a safe and deterministic environment without interacting with the main Ethereum blockchain.
+
+### Which programming languages for blockchain?
+
+1. Solidity
+
+This is the programming language that is used for creating Ethereum smart contracts.
```

Comment:
This is probably the language that will definitely catch the eye of a reader. Perhaps provide a link to the Solidity page or a useful resource.